src/Dropdown.elm

```diff
@@ -416,22 +416,26 @@ updateKeyDown config key model state =
     in
     case key of
         ArrowDown ->
-            ( State { state | focusedIndex = clampIndex <| state.focusedIndex + 1 }, Cmd.none )
+            ( State { state | isOpen = True, focusedIndex = clampIndex <| state.focusedIndex + 1 }, Cmd.none )
 
         ArrowUp ->
-            ( State { state | focusedIndex = clampIndex <| state.focusedIndex - 1 }, Cmd.none )
+            ( State { state | isOpen = True, focusedIndex = clampIndex <| state.focusedIndex - 1 }, Cmd.none )
 
         Enter ->
-            ( State { state | isOpen = False }
-            , case findFocusedItem config.itemToText state.filterText state.focusedIndex items of
-                Just focusedItem ->
-                    selectedItemsAsList config model
-                        |> modifySelectedItems config.dropdownType focusedItem
-                        |> updateSelectedItemsCommand config.onSelectMsg
+            if not state.isOpen then
```

Comment:
yeah, that's it, you can just simplify by removing the "not" and inverting the branches.

Reply:
In an `if then else`, I prefer to put the short block right after the `if`, so for the `else`, you do not have to scroll up to find what the `else` is about.
If I use `if state.isOpen then`, we get a longer block, and the `else` is all alone by itself, if you know what I mean.

Generally thought, I avoid booleans altogether and use case statements. Here, that would mean that we would have a state.visibility which could be Hidden or Visible. Then it would become
`case state.visibility of
  Hidden -> ...
  Visible -> ...`
I am not sure if you want me to go this far.